docs/general/contributors.md

```diff
@@ -13,11 +13,13 @@ is listed as a precaution - if one of these people ever contacts you directly, f
 information to see that it matches the origin of the message, and then feel free to get in touch with someone 
 else on this list to verify the legitimacy of the original inquiry. 
 
-:warning: **Employee impersonation is a common type of scam**. :warning:
+:::warning Employee impersonation is a common type of scam
 
-> For other ways of protecting yourself, please see [How to Protect Yourself from Scams](scams.md).
+For other ways of protecting yourself, please see the [How to Protect Yourself from Scams](scams.md) guide.
 
----
+:::
+
+## Technical Education
```

Comment:
The reorganization and changes to the Contributors page need to be in a different PR and will need separate review pipeline.

Reply:
https://github.com/w3f/polkadot-wiki/pull/3168